src/Microsoft.AspNet.Server.Kestrel/Http/SocketInput.cs

```diff
@@ -238,6 +239,10 @@ public void GetResult()
             var error = _awaitableError;
             if (error != null)
             {
+                if (error is TaskCanceledException || error is InvalidOperationException)
```

Comment:
@Tratcher I would just like a second opinion on changing the `IOException` to `TaskCanceledExceptions` when the request is aborted. I think it's fine.